task-sdk/src/airflow/sdk/execution_time/task_runner.py

```diff
@@ -385,16 +385,14 @@ def get_first_reschedule_date(self, context: Context) -> datetime | None:
         if TYPE_CHECKING:
             assert isinstance(response, TaskRescheduleStartDate)
 
-        start_date = response.start_date
-        log.debug("First reschedule date from supervisor: %s", start_date)
-
-        return start_date
+        return response.start_date
```

Comment:
Can we keep the log?

Reply:
The date can be `None` too so I have removed it. Might add it back in related changes but since CI is green I would like to merge it